dependency-graph/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Dependency Review",
+    "creator": "GitHub",
```

Comment:
No need to have creator if you want to show up as `By GitHub`.

Reply:
Excellent, can update 👍